dll/shellext/sendmail/lang/ro-RO.rc

```diff
@@ -0,0 +1,14 @@
+/*
+ * PROJECT:     ReactOS Send Mail
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2022 Andrei Miloiu <miloiuandrei@gmail.com>
+ */
+
+LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+{
+    IDS_DESKLINK, "Birou (Creare comandă rapidă)"
+    IDS_SHORTCUT, "Comandă rapidă la %s"
+}
```

Comment:
Same for this one.

Reply:
So, what should I do with these two files? Shouldn't they be placed in lang folder?

Reply:
Or the problem is the header?

Reply:
The same as I explained earlier: https://github.com/reactos/reactos/pull/4693#issuecomment-1247355524

Reply:
Ok, I will edit the files.